### ISSUE TITLE
Pass-alive stones can die according to territory estimator

### ISSUE THREAD

**Issue by pdeblanc** [lightvector/KataGo#773]
From Kellin Pelrine's 2023-01-22 game here:

https://goattack.far.ai/human-evaluation#human_vs_kata100k

I looked at this game with my local copy of KaTrain. I was surprised to see the N15 pass-alive group evaluated as probably dead:

<img width="720" alt="スクリーンショット 2023-03-15 0 10 42" src="https://user-images.githubusercontent.com/6886508/225046417-b9bac20f-42a4-4cbe-b49a-a3026cebcb01.png">

This is with the `b40c256-s11101799168-d2715431527` network.

It is my understanding that there is no situation in which a pass-alive group will die given reasonable play, and that pass-aliveness is one of the features provided as an input to the model.

Not sure how this would be resolved, although I can imagine a couple possibilities:
- Set a training target for ownership in the case of pass-alive groups
- Patch the territory estimate for pass-alive groups after the network evaluation

**Comment by lightvector** [comment]
Yep, this has been noticed before! I don't think there is any necessary action to resolve this.

> It is my understanding that there is no situation in which a pass-alive group will die given reasonable play, and that pass-aliveness is one of the features provided as an input to the model.

Pass-aliveness is *not* an input to the model. Current area ownership if the game were scored right now is an input to the model, which may include territories that contain opponent stones if those opponent stones are deemed pass-dead and the surrounding group pass-alive. However, this usually provides little information to the model about pass-aliveness, for example usually a dead one-eyed group will not be pass-dead and therefore will be counted as current area ownership if the game were scored right now. So mostly the neural net will be computing liveness on its own.

The ownership target's weight in the loss function is not very large, and even older nets seem to have some idea that cyclic groups with a very small cycles are "weird" in their value, although it doesn't play them well at all, so possibly what happened is that the net adjusted internal activations in this highly unusual kind of cyclic group case in a way that resulted in better value/policy/other target predictions in the training data, and these activations just so happened to result in the ownership target marking the group as dead, and the ownership target hasn't had enough training with cyclic groups to learn or fight back or readapt to predict the correct ownership of the internal group when those activations look like that.

This seems like the sort of thing that should eventually fix itself with more training, especially once the density of cyclic group positions is increased. Would you be interested to try more recent nets which have started to be augmented with cyclic groups in the training data?
